packages/mui-material-next/src/CircularProgress/CircularProgress.test.tsx

```diff
@@ -53,21 +62,27 @@ describe('<CircularProgress />', () => {
     const circularProgress = getByRole('progressbar');
     const circle = container.querySelector<SVGCircleElement>('circle');
     expect(circularProgress).to.have.class(classes.root);
-    expect(circle).to.have.class(classes.circleIndeterminate);
+    expect(circularProgress).to.have.class(classes.indeterminate);
+    expect(circle).to.have.class(classes.circle);
   });
 
-  it('should render with a different size', () => {
+  it('should render with a different size', async function test() {
+    if (/jsdom/.test(window.navigator.userAgent)) {
+      // only run on browser, because getComputedStyle only works in browser
+      this.skip();
+    }
     const { container, getByRole } = render(<CircularProgress size={60} />);
     const circularProgress = getByRole('progressbar');
+    const circularProgressStyle = window.getComputedStyle(circularProgress);
     expect(circularProgress).to.have.class(classes.root);
-    expect(circularProgress.style.width).to.equal('60px', 'should have width correctly set');
-    expect(circularProgress.style.height).to.equal('60px', 'should have height correctly set');
+    expect(circularProgressStyle.width).to.equal('60px', 'should have width correctly set');
```

Comment:
Does `circularProgress.style` not work anymore?

Reply:
@DiegoAndai In principle yes, but not in typescript, so I had changed this.